wdl/ApplyHardFilter.wdl

```diff
@@ -0,0 +1,81 @@
+version 1.0
+
+import "Structs.wdl"
+
+workflow ApplyHardFilter {
+  input {
+    String prefix
+    File vcf
+    String? filter_name
+
+    String sv_base_mini_docker
+    RuntimeAttr? runtime_attr_hard_filter_wham
+  }
+
+  call HardFilterWham {
+    input:
+      prefix = prefix,
+      vcf = vcf,
+      filter_name = filter_name,
+      sv_base_mini_docker = sv_base_mini_docker,
+      runtime_attr_override = runtime_attr_hard_filter_wham
+  }
+
+  output {
+    File hard_filtered_vcf = HardFilterWham.hard_filtered_vcf
+    File hard_filtered_vcf_index = HardFilterWham.hard_filtered_vcf_index
+  }
+}
+
+
+# Removes Wham-only, SR-only deletions <1kb
+task HardFilterWham {
+  input {
+    String prefix
+    File vcf
+    String? filter_name
+
+    String sv_base_mini_docker
+    RuntimeAttr? runtime_attr_hard_filter_wham
+  }
+
+  String filter_name_ = select_first([filter_name, "filter_wham_del_sr_1kb"])
+  String hard_filtered_vcf_name = "~{prefix}.~{filter_name_}.vcf.gz"
+
+  # Disk must be scaled proportionally to the size of the VCF
+  Float input_size = size(vcf, "GiB")
+  RuntimeAttr default_attr = object {
+    mem_gb: 3.75,
+    disk_gb: ceil(10.0 + (input_size * 1.5)),
+    cpu_cores: 1,
+    preemptible_tries: 3,
+    max_retries: 1,
+    boot_disk_gb: 10
+  }
+  RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
+
+  command <<<
+
+    set -euo pipefail
+
+    bcftools view -e 'SVTYPE=="DEL" && COUNT(ALGORITHMS)==1 && ALGORITHMS=="wham" && SVLEN<1000 && COUNT(EVIDENCE)==1 && EVIDENCE=="SR"' ~{vcf} -Oz -o "~{hard_filtered_vcf_name}"
```

Comment:
Let's make this workflow reusable and let the user provide the filtering criteria as a String:
```suggestion
    bcftools view -e '~{filter}' ~{vcf} -Oz -o "~{hard_filtered_vcf_name}"
```